nomad/leader.go

```diff
@@ -458,6 +470,25 @@ func (s *Server) reconcileMember(member serf.Member) error {
 	return nil
 }
 
+// reconcileJobSummaries reconciles the summaries of all the jobs registered in
+// the system
+// COMPAT 0.4 -> 0.4.1
+func (s *Server) reconcileJobSummaries() error {
+	index, err := s.fsm.state.LatestIndex()
```

Comment:
The debug message below would indicate the new leader has reconciled the summaries. 

There is no way to differentiate between reconciliation by user and leader by just looking at fsm logs.